choco/tools/chocolateyinstall.ps1

```diff
@@ -9,14 +9,23 @@ $packageArgs = @{
   checksumType  = 'md5'
 }
 
-Install-ChocolateyZipPackage @packageArgs
+# get Package Parameters
+$pp = Get-PackageParameters
 
+Install-ChocolateyZipPackage @packageArgs
 
 Write-Host "Running tlmgr path add"
 # Adds to Path
 $statementsToRun = "/C `"$toolsDir\TinyTeX\bin\win32\tlmgr.bat path add`""
 Start-ChocolateyProcessAsAdmin $statementsToRun "cmd.exe"
 
+# checks if choco switch AddToSystemPath var is in parameters
+if ($pp['AddToSystemPath'] -eq 'true') {
+   Write-Host "Running tlmgr AddToSystemPath"
+   # AddToSystemPath
+   $statementsToRun = "/C `"$toolsDir\TinyTeX\bin\win32\tlmgr.bat path --w32mode=admin add`""
+   Start-ChocolateyProcessAsAdmin $statementsToRun "cmd.exe"
+}
```

Comment:
If we were to do that, I think we just need to add the flag in the previous line when the option is passed to chocolatey. 

https://github.com/rstudio/tinytex-releases/pull/24/files#diff-5a71f76f6ddf465237c7306d0e59eb56e11db5cfb7281b4a5a647306dda6efe5R19-R20

Here we are putting tlmgr twice in the PATH. First user, then system. It does not seem right.

Reply:
combined the two lines, but I am not sure if this is the way to do it.
At least it works with the if operator.